python/lsst/cp/pipe/cpCombine.py

```diff
@@ -341,6 +341,18 @@ def run(self, inputExpHandles, inputScales=None, inputDims=None):
         self.combineHeaders(inputExpHandles, combinedExp,
                             calibType=self.config.calibrationType, scales=expScales)
 
+        # Populate a visitInfo.  Set the exposure time and dark time
+        # to 0.0 or 1.0 as appropriate, and copy the instrument name
+        # from one of the inputs.
+        expTime = 1.0
```

Comment:
How come the exposure time isn't the sum of all the dark exposure times? We could easily calculate that in combineHeaders since we already loop over visitInfo to store the exposure times in the header.

Reply:
It's 1.0s as the scale factors used to combine the input darks are the `dark_time` values.  This normalizes the combined dark to have unit exposure time, which is then scaled to match the `dark_time` of the exposures that will be corrected (in `IsrTask`).